### ISSUE TITLE
Premade Issue Post for those who need it.

### ISSUE THREAD

**Issue by jewbmx** [jewbmx/repo#32]

**Comment by axxell6307** [comment]
Excellent work! Scrubs is working great. I was curious if you have any plans to change the addon icon? 😁

**Comment by jewbmx** [comment]
Lol why would i?

**Comment by axxell6307** [comment]
Ok let me stop being an ingrate. Lol Thanks for the great support. Love the dup checker option also.

**Comment by jewbmx** [comment]
Lol your welcome and thanks back, dupe checks thanks to doko. Found 2 ive had sitting in my modules for months tho i didnt know about that i just used in a new scrape module that should keep a decent amount of the dupes out before they are counted up.
Probably updating tomorrow to get the changes shared.

**Comment by JaxJoe01** [comment]
Movies work great, but tv shows are hit or miss.  Torrents whether it is set to 1 seed avail or 20, they skim through all and some shows come up with no streams available.  I posted a month or two ago which sites didn't work even though they have the shows and streams avail.  You did fix rsb but others still come back with none or some shows from the scrapes.

Went through your other posts looking for ways to help out or help modify some of the site scrapes but there is not much info to go on to assist.  So I am sorry for lack of helping out on that part.

**Comment by jewbmx** [comment]
Movies is still slightly bad because a couple scrapers are a pain in the ass for a year check or simply dont have it at all.  Tv shows is mostly ass about 40% of the hosters or sites themselves have the wrong episode, then theres also a few left that just need code updates to tighten its results.  Torrents is simply because the seeder setting is only in a couple of the scrapers, then the next issue on em is if your paid service is actually actively caching the files or has a low user base. Maybe cache your own files like its supposed to be lol.

As for your previous posts they offered little to no info so they got tested and dealt with as i seen fit, if they are still here its because they work fine for me.

And last but not least, you dont gotta help me out in any ways unless actually wanting to lol, i pretty much just do my stuff for fun and to enjoy the day. The only real goal or objective is to shine a little light on how easy it all is and maybe help people do it their self lol.

**Comment by jewbmx** [comment]
Oh yea also thats my bad on the lack of posts now, i was on the laptop the other day and seen the option to delete stuff so i went a little clean happy lol

**Comment by jewbmx** [comment]
Quick warning for anyone that enables this new scraper/provider i just made labeled  gomoviesonl, its a nice super scraper that will blast you with 1000 results of all qualities with most the main hosters lol. Downfall to it is they dont clean the nojoys out. If ya use it id suggest the host limit setting, dupe check only knocks about 20 off. If ya use it for shows only the results are alot cleaner.

**Comment by Tinuz97** [comment]
MkvHub broken? (not returning sources ) or maybey it is just me,no idea, just wanted to 'report' it.

**Comment by brianlee1961** [comment]
Since last update search does not get link's for older TV shows ie law and order nypd blue was working other day now nothing's showing up

**Comment by jewbmx** [comment]
Looks like mkvhub did some changes to their site so the scraper will probably work again if i update it or rewrite it.

As for older shows not showing up if you use dbird services like real-debrid or PM you can find almost anything with the torrent scrapers on. If your a free stream type like me then id suggest you enable all the normals and spares, run a search or two and keep track of what ones are bringin results, then pop back into the settings and disable the ones ya dont like again besides the ones you got results with.  Depending on the show season and your location, you should have about 4 'free' scrapers that work on old L&O.

I havent really removed anything good lately so you might just need to clear sources and pick your new favs.  Was hoping the 20ish new scrapers i made this month would help get older stuff but it looks like i gotta keep trying lol.

**Comment by jewbmx** [comment]
If either of you are looking for a quick fix to your problems you might have some luck with Tempest, the addons direct type of scrapers might have your episodes,  and he might of already fixed that scraper you are not getting results from.

**Comment by brianlee1961** [comment]
Cheers will have a look by the way I use your addon on a regular basis
thanks for all your hard work

**Comment by brianlee1961** [comment]
Cheers new update has enabled the search function ie search for older TV
shows ie law and order nypd blue to work again many thanks

**Comment by phill-nz** [comment]
hi .. how do i change viewtypes to media info .. i dont seem to be able to change it at all ... what am i doing wrong 
and big up's for the scrubs v2 .. best i have used in a long time 

cheers
phill

**Comment by jewbmx** [comment]
my bad phill just seen this problem ya got lol, if your still having issues you can reply with some details and i can look into it.

now for my reason of hoppin on here lol.  (Temp Link Drop} https://gamovideo.com/embed-ac581aie4xt6.html

**Comment by phill-nz** [comment]
hi
i hit on confluence ( still the easiest and quickest to use ) 
videos / scrubsv2 / tools / scrubs ; viewtypes ...
 it just comes up as a rectangle with the headers on it ( movies, tv shows, seasons, episodes, ) but clicking on any gives no link to change it , it just blanks to scrubs ; viewtypes again 
thanks for the reply .. i do much prefer media info for movies / tv shows and episodes pages for seasons thumbs are fine 

great work maintaining this 
again 
thanks

**Comment by jewbmx** [comment]
K looks like i put a isFolder=False on that option thinking it was always a dialogbox lol. If you open the Navigator Indexer and remove that bit it will work before i update again. (Can also open it and change the false to true lol.)

I never touch that option so im not sure how it normally works, but it seems to be all good now.

**Comment by jewbmx** [comment]
If anyone has any unaddressed issues/errors ya might wanna hit me up before my next update, because its the last one for a bit. Imma update my own devices and not care lol. Got other things imma work on instead of holding limp dev hands.

**Comment by warmach1ne23** [comment]
> K looks like i put a isFolder=False on that option thinking it was always a dialogbox lol. If you open the Navigator Indexer and remove that bit it will work before i update again. (Can also open it and change the false to true lol.)
> 
> I never touch that option so im not sure how it normally works, but it seems to be all good now.
> 
> (Also if anyone is curious on fixing that gamovideo shit the problem is just http vs. https. The final url needs to always be http otherwise it shits the bed.)

Where do I find the navigator indexer to edit?

**Comment by jewbmx** [comment]
That all depends on your setup. For me on my phone its file:///storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/script.module.scrubsv2/lib/resources/lib/indexers/navigator.py

Then towards the bottom of the file if i recall should be the def tools  and the line will say view in it lol.

**Comment by warmach1ne23** [comment]
> That all depends on your setup. For me on my phone its file:///storage/emulated/0/Android/data/org.xbmc.kodi/files/.kodi/addons/script.module.scrubsv2/lib/resources/lib/indexers/navigator.py
> 
> Then towards the bottom of the file if i recall should be the def tools and the line will say view in it lol.

K thanks found & fix it. It's been bothering me for days now.

**Comment by jewbmx** [comment]
Np ive still never used the option so glad to have it tested too lol.

**Comment by jewbmx** [comment]
Uploaded the update to fix everyones issues last night, feel free to hit me up again if i missed anyones issue or if ya come across another.

Unless contacted to fix something, the next update should be around the end of the month. Any issues until then will be fixed and uploaded asap since my addon will still be sitting here cleaned from this update lol. Im gonna be working on a different addon while the scraper pile dies so my advice is disable the ones with delays til im back at it, or hit me up in a email for that scrapers fix.  Im around and available about 19 hours a day every day so dont worry about time ;)

**Comment by phill-nz** [comment]
thanks for fixing that 
works well now

**Comment by jewbmx** [comment]
On a hunt for soap opera sites or sites with soaps, if anyone wants to assist or got some lol.  Yesmoviegg seems to be one of the only ones that work for em. Young and the restless is the search item

**Comment by phill-nz** [comment]
hi
if you feel like doing some adding and scrubtracting 
some ideas ...

a manual "watched episode" 'click box' as the auto one is often not working ( all xbmc derivatives ) 

a "last used" tick box on the provider for episodes .. so we can remember the best ones that worked for that series ( maybe even a sort of providers for "most used links" ) 

i probably have plenty of others . but im testing the water with these 
fly / shot down / sunk

**Comment by bigpoppapaul** [comment]
Hey man, first let me say I love Scrubs V2 and it's my main go to addon. It's lightweight and lightening fast and you maintain the scrapers incredibly well. So thank you. 

I do have a quick question about an issue I'm having. It's really not THAT big a deal, but I thought I'd ask anyway. 
Sometimes, some Trakt lists don't populate properly with thumbnails/fanart. I have my own apis for fanart, tmdb, and omdb. I've tried clearing meta cache on Scrubs and clearing thumbs in Kodi. To no avail. In comparison, the same lists in Venom with the same apis attached pull all thumbnails. 

As an example, here is Scrubs for [Gilad&Alex's "Latest Releases" Trakt list.](https://trakt.tv/users/giladg/lists/latest-releases?sort=added,asc)
![Scrubs](https://user-images.githubusercontent.com/20094991/64859515-700cbf00-d5f8-11e9-87c9-effb501ffad9.png)

And here is Venom with same list. 
![Venom](https://user-images.githubusercontent.com/20094991/64859544-89157000-d5f8-11e9-8af8-9814f255f200.png)

Some areas of the lists pull the thumbnails no problem, and then other areas I get chunks like this where Scrubs has problems doing it. 

Any suggestions on why this might be happening and how I can resolve it? I'm on Kodi 17.6 with Xonfluence skin.

**Comment by jewbmx** [comment]
When it comes to Kodi versions im only using 18.4 on all my devices so older versions are pretty much abandoned besides the basic issues people hit me up about like cfscrape updates that were made for 18 and error on 17 lol.

But your issue is most likely just a code issue since ive never cared to work on the "superinfo" bit of the addon. Venom is actually one of the only exo/cov forks around that has taken the time to work on that big bit of code, the dev spent alot of time and effort on it a while back when we were both in a chat geekin out on random issues. I played with it a little but didnt like the different artwork the changes gave me so i ditched it all lol.

I rock everything basically stock.. kodi build, aeon nox silvo skin, and addon so i dont really ever have much artwork missing for the most part. I actually forgot all about trakt lists so i might dig thru my phone and find all of the ones ive been sitting on since bearded bandit and while im at it i will try to fix the superinfo up a little for ya too. No pics makes it pretty shitty to find a flick lol.

**Comment by jewbmx** [comment]
I just remembered that i have a couple trakt lists i made so i can be lazy and skip my search for em lol. Also reminds me that my dev menus test show list is broken for some reason so i gotta try to find out why as well before i forget again lmao

**Comment by bigpoppapaul** [comment]
Thank you for the quick response. And yes, the artwork is there consistently in your addon, except for the Trakt lists. Aside from watchlist and collections on my own account, the only other Trakt list I use is the one I linked for you simply because it's meticulously maintained and keeps track of all new HD/Bluray/etc. releases. 
Anyways, like I said, not a huge deal if it's backbreaking (fingerbreaking?) labor, but I thought I'd ask and see what you replied with.  

Thanks again for your addon. :)

**Comment by jewbmx** [comment]
I fixed all the TMDb UserList stuff up a bit and decided to hold off on fixing the superinfo and all that stuff for now.

TMDb is moving on to a v4 so i am gonna wait a few months and see if they put more into it. (Right now v4 is only really up and running for auth and list stuff. While all the other stuff is still just v3.)

**Comment by bigpoppapaul** [comment]
Alright man, thanks for the update. I really appreciate it. :)

**Comment by jewbmx** [comment]
Gonna be posting a real update in a couple hours at most. Its step 1 of my remodel lol just gonna finish a bit more testing

**Comment by bigpoppapaul** [comment]
What kinds of things are you doing with the remodel?

**Comment by jewbmx** [comment]
Some random changes and cleanup, some language work, multiple navigation changes, probably a few setting changes, and most likely gonna add in the My TMDb sections or prep for em to get added later. Lol theres quite a list made up and saved up.

**Comment by bigpoppapaul** [comment]
Looks good to me. Thanks for starting on the TMDB stuff. 

Here's a small feature request that for some reason I missed before. How about a "disable all foreign providers" toggle? You have it for normal, debrid and torrents already, the only one missing is foreign. :)

**Comment by jewbmx** [comment]
Yea i skipped those so i wouldnt have to mess with em lol, the setting uses a list of providers i make to know whats what and i was just being lazy for foreign. You can do the process tho if you go to my dev settings and hit disable all. Its by the debug settings lol but then you gotta double check em and enable the ones ya want again.

I will go ahead and make the settings and their bits for ya tho since im already making changes.

**Comment by bigpoppapaul** [comment]
Oh, I didn't even NOTICE that. Don't think there is a need for further changes then. I can just disable all and then switch on the others like you said. 
Thanks for the heads up.

**Comment by jewbmx** [comment]
Incase anyone reads this that didnt notice my upload today, i have wiped out the old language files and replaced em with a clean new version thats not translated yet.

If you use another language in Scrubs then let me know and i will translate it before i upload the next update. Or you can do it in the new repo i made and ill add it in.  Either way sorry for those that used non-english lol a clean start was easier :)

**Comment by jewbmx** [comment]
I am at a good stopping point on my changes and cant recall what all i got left so imma upload the new update tonight lol.
Whoever had the Trakt list issue see if i fixed it lol i dont got trakt on this device. Also i decided to not do the enable/disable settings for the rest of the providers, its on hold for now so i can try and test em all to see what works that way im not coding more broken junk into the addon.

**Comment by bigpoppapaul** [comment]
Don't worry about the enable/disable settings for the rest of the providers. Once you pointed out the "disable all" in dev settings, that's all I needed. 
I'm also the guy that brought up the Trakt thumbnail (superinfo) issue. I'll let you know for sure. 
Thank you.

**Comment by jewbmx** [comment]
Lol that makes things easy. Imma do the settings anyways because it'll make things easy. 

Updates should be poppin up in the next 5 mins or so because i just uploaded.

Two things i probably forgot to say in the notes is that the metadata module can be removed now making the addon alot lighter and quicker on a fresh install.
And the second is just a warning that when you goto add custom userlists in the new settings, make sure you save before switching tabs, i noticed it when setting up for tests lol.

**Comment by Tinuz97** [comment]
I see a little typo in the scraping screen, Dbird instead of Debrid 😅

**Comment by jewbmx** [comment]
Thats not a typo lol its my way of making fun of debrid i always call it dbird

**Comment by Tinuz97** [comment]
Ah, that is why it is faster with the new update, it is flying like a bird now 🐦😄

**Comment by bigpoppapaul** [comment]
Just thought I'd let you know that the Trakt list thumbnail problem is still there. To be clear, I'm not complaining as the addon still works great, I'm only letting you know because you asked me to let you know. :) 

![SPMC_20190928_213306](https://user-images.githubusercontent.com/20094991/65824479-f69be000-e237-11e9-868b-3132cce7fa90.png)

**Comment by jewbmx** [comment]
I just spent a good chunk of the morning trying to get my device to show my scrubs artwork like your setup paul and couldnt so i am not real positive about getting that list issue fixed lol. Im totally done swapping skins around as well, thats a headache and just straight weird being off silvo lmao.

**Comment by jewbmx** [comment]
Just pushed a little module update to fix the ustvgo plug and pretty sure the artwork got changed up too in movies but i cant remember if i left any of the changes or not lol

**Comment by jewbmx** [comment]
I wont be doing another update for a lil bit since im already in the middle of some stuff but if anyone needs their usTVgo Indexer updated i just uploaded the py file to my ScrapeScraps repo so you can toss it in yourself.  I was just checking for new channels and found they updated the pages and the changes broke the Indexer your sitting on ;)  New ones working proper and has the new channels added as well.

**Comment by jewbmx** [comment]
If anyone is advanced or skilled enough to add in a new custom indexer... i am working on a new one for watchwrestling.la and watchwrestling.cz that is gonna need some good testing done lol. If your interested or wanna help fix it up feel free to email me, otherwise you can do the testing after i push another update with it completed as a v1

**Comment by bigpoppapaul** [comment]
> I just spent a good chunk of the morning trying to get my device to show my scrubs artwork like your setup paul and couldnt so i am not real positive about getting that list issue fixed lol. Im totally done swapping skins around as well, thats a headache and just straight weird being off silvo lmao.

Not a problem man.  Thanks for trying anyway.

**Comment by jewbmx** [comment]
Dropped a new update with the new wrestling section, some sources wont work til the next resolveurl update, also alot of em get 404 errors due to copyright bullshit altho i tried to block the most common hosts that get picked off.

**Comment by 123Venom** [comment]
artwork is an easy fix..look close, key, log, check.  What did you say to me last time shoe was on other foot, shit[?} something like that anyway..lol

**Comment by jewbmx** [comment]
I am no longer doing updates to anything here. From now on if you wanna update you can email me personally or stick with this version. Beware of future fakes lol.

Yet again people are taking my work and publishing it to hundreds and hundreds of more people so im bowing out. Sites have enough trouble without me being a cause for their demise.

**Comment by phill-nz** [comment]
nooooo
or to put it plainly 
this is the best and far to good to abandon 

i can see your point but some need to stay and steer the ship

**Comment by phill-nz** [comment]
ohh yehh
and i cant thank you enough for what you have done already .. it really is a pleasure using this one

**Comment by axxell6307** [comment]
> I am no longer doing updates to anything here. From now on if you wanna update you can email me personally or stick with this version. Beware of future fakes lol.
> 
> Yet again people are taking my work and publishing it to hundreds and hundreds of more people so im bowing out. Sites have enough trouble without me being a cause for their demise.

You're retiring scrubsv2? Say it isn't so 😭. One of the GOATs of kodi. Gotta do what you gotta do though. I understand. 😔

**Comment by bigpoppapaul** [comment]
Awww man. I LOVE your addon. Sorry to see you bow out but thank you for everything you have done. 

What is this about emailing you personally for updates? Was that sarcastic or is that something you're willing to do?

**Comment by axxell6307** [comment]
> I am no longer doing updates to anything here. From now on if you wanna update you can email me personally or stick with this version. Beware of future fakes lol.
> 
> Yet again people are taking my work and publishing it to hundreds and hundreds of more people so im bowing out. Sites have enough trouble without me being a cause for their demise.

What are the current fakes, if you can say? Can we really email you for updates? haha

**Comment by bjgood** [comment]
Hate to see you closing shop, JewBMX. You're one of the very few who keep this hobby of ours going. 

Wish you would change your mind on this decision, dude.

Best wishes.

**Comment by Dustychonchon** [comment]
> I am no longer doing updates to anything here. From now on if you wanna update you can email me personally or stick with this version. Beware of future fakes lol.
> 
> Yet again people are taking my work and publishing it to hundreds and hundreds of more people so im bowing out. Sites have enough trouble without me being a cause for their demise.

Man this sucks huge balls. Your 24/7 channel and sports sections are top notch and get me thru my day. I'll be sad to see you go but Ive been using kodi for years and the greats always seem to leave. sucks ive seen many come and many go

**Comment by Tempest0580** [comment]
This is the same reason it's so hard to stay motivated to keep tempest updated. Basically I've only been doing updates as needed for my personal use.

**Comment by RickFromTexas** [comment]
Tempest0580, just wanted to thank you and jewbmx, the work you guys accomplished kept the world of Kodi sane for a while. It might not be enough motivation but you should feel very proud of the work you did. 

The reason people copy your work so much is because you did such **awesome** work!

**Comment by Tempest0580** [comment]
Thanks Rick.  I haven't given up yet. I'm still going to be doing updates. I'm going to be adding a bunch of 1 click movies and tv shows to tempest and temptv.  For now they will be debrid only until I can find some good reliable free sources.

**Comment by RickFromTexas** [comment]
Glad to hear you'll be around for a while. I use Tempest, TempTV, or Scrubs every day so hearing that jewbmx was leaving and that you're having trouble staying motivated got me worried. After XC, then streamango & openload, and now losing so many other hosters, it's a tough time to be a Kodi user.

But thanks for getting back to me, I just wanted to make sure you guys knew how much the work you did was appreciated. Kodi users keep a low profile and don't always let developers know how much their work is appreciated but it really, really is. 

Thanks again for everything!

**Comment by bjgood** [comment]
Nicely put, RickFromTexas. I agree. Both Tempest and Jewbmx are good guys. I like them.

**Comment by jewbmx** [comment]
Still not taking the time to reply to anyone yet but figured i should atleast let everyone know i will upload a minor update in a few days at most fixing the tvdb api changes. Just waiting to see if any last minute changes are made on their end.

**Comment by bigpoppapaul** [comment]
That is VERY kind of you. Thank you!

**Comment by jewbmx** [comment]
Might need a few more testers to hit me back but its looking like no updates even needed, everything seems to be back to normal for everyone so far

**Comment by bigpoppapaul** [comment]
Shit, really? I'm still definitely having the TVDB problem on multiple devices. Had no idea others had the issue resolved.

**Comment by jewbmx** [comment]
Lol i stopped typing halfway thru that msg to do a dab and forgot a few words in it lmao. Supposed to say no other updates are even needed, meaning what i fixed and sent to a few buddies appears to be all good.

**Comment by bjgood** [comment]
swatchseries.to is screwed. Doesn't provide any links (at least the last time I tried...two days ago...tvdb).

**Comment by jewbmx** [comment]
Lmao for that level of ignorance imma wait till next weekend to upload the tvdb update. I give no fucks for your scraper troubles

**Comment by bjgood** [comment]
I really don't understand your level of hostility towards me, Jewbmx. It's puzzling.

Mind to elaborate why?

> Lmao for that level of ignorance imma wait till next weekend to upload the tvdb update. I give no fucks for your scraper troubles

**Comment by Dustychonchon** [comment]
> I really don't understand your level of hostility towards me, Jewbmx. It's puzzling.
> 
> Mind to elaborate why?
> 
> > Lmao for that level of ignorance imma wait till next weekend to upload the tvdb update. I give no fucks for your scraper troubles

well he just said hes pretty much quitting and youre asking him to continue to give updates to things

**Comment by bigpoppapaul** [comment]
> Lol i stopped typing halfway thru that msg to do a dab and forgot a few words in it lmao. Supposed to say no other updates are even needed, meaning what i fixed and sent to a few buddies appears to be all good.

Yeah, I quickly went and checked multiple devices and addons after you sent that last one thinking I'd missed something. haha

**Comment by bigpoppapaul** [comment]
> Might need a few more testers to hit me back but its looking like no updates even needed, everything seems to be back to normal for everyone so far

If you need anymore testers, feel free to let me know. I'd be happy to.

**Comment by bigpoppapaul** [comment]
Thanks for the update @jewbmx. Really appreciate it.

**Comment by jewbmx** [comment]
Just a quick fix to make the shows work again. Still gotta wait and see if they ever fix up their changes to tvdb since it causes about 4 different issues that ive seen.

Also dont mind tho because im on a movie kick and been working on a new nitecrawler update. Downfall there is its a 3way fuckfest lol between main addon, scraper module, and a custom resolver script that all need some lovin

**Comment by bigpoppapaul** [comment]
Yeah, but it did fix a lot of the TVDB stuff. I'd cobbled together a small fix in episodes.py just to make the shows work again, but I wasn't getting any thumbnail art, but your update fixed that for the most part. So I'm grateful. 

I haven't even played with your nitecrawler addon because I use Scrubs for everything (with Venom and Seren as backups). I may have to check it out.

**Comment by jewbmx** [comment]
Honestly if you dont got it yet probably wanna wait for the update to be uploaded to prevent any errors or issues my changes might make lol. Its just scrubs with universalscrapers pretty much.  Plan was finish its update while scrubs dies off scraperwise but now imma work a little quicker due to all the random funk that keeps sprouting up

**Comment by bigpoppapaul** [comment]
Out of curiosity, what is the advantage, if any, of Nitecrawler over Scrubs for shows/movies? Any advantage to Universal Scrapers for... say... older/rarer content?

**Comment by jewbmx** [comment]
Well codewise its pretty much the exact same, and all the same issues. Scraperwise i couldnt say because i havent played anything with it for like 6 months lol but most the sites are in both addons or another scraper that matchs up to it but scrubs has more so its likely scrubs would find more items, also final links are checked better in universal scrapers if i recall with the check bit.

**Comment by bigpoppapaul** [comment]
Ok, thanks man, good to know. I'll check it out once you do the newest updates to it.

**Comment by jewbmx** [comment]
Welp 4 hours later i figure out one tiny line i missed that fixes the speed and artwork lol. Havent fully tested it but dont care either way lmao works good enough for me, ill upload it for yall in scrubs after a few minutes so i can get my high back

**Comment by jewbmx** [comment]
If anyone is bored and wants to help out with some resolver stuff, i gotta make a list of all the resolvers aka hosts that dont work. Dont need sample urls because im sure i can find one on my own lol.

Game plans to gather em all up and since the most common issue is cfscrape im gonna be making a custom resolver for em that will try to do the job once resolveurl fails.

If you happen to code and wanna help make em you can email me or whatnot and i can send you some sample code to see the structure/framework.

This is the chore for the week and bump in my path so imma be at it before i do any other stuff lol

**Comment by jewbmx** [comment]
Not sure who Algirdas is but your money you sent to my paypal should be auto returned when the time runs out for me to accept it lol, think its 20 or 30 days.

I dont actually use paypal or money but thanks for the thought :) its a nice gesture

**Comment by jewbmx** [comment]
Ghetto rigged channels.py fix if you are skilled enough to add it yourself. (Fixes artwork.)  https://textuploader.com/1ooda/raw

**Comment by bigpoppapaul** [comment]
Thanks man!

**Comment by jewbmx** [comment]
Seems i gotta tighten up one of the quality codes used for scrapers, random links with the right numbers lined up are being marked as 4k and so on when they arent. Like the dragon prince season 3 showing up as 4k fembed when they are actually just 1080p results lol

**Comment by jewbmx** [comment]
TVDb is finally fixing itself up so errors are likely with the latest updates ive put here. Id suggest trying to go back a version or two if ya need to. (I wont be pushing an update for a while.)

Oh and on a side note, if someone wants to contact me that doesnt speak good English dont bother trying. Just speak to me in your own language and i will translate it if i need to. I already do it with a couple people that speak true gibberish that they call english lol

**Comment by jewbmx** [comment]
Uploaded a small update for the tvdb changes. If you used the sky channels fix then you will need to reapply it. I dont think i added it in to scrubs.  If i recall the only thing changed is tvdb related

**Comment by jewbmx** [comment]
If ya need a ustvgo fix here ya be   https://textuploader.com/1oudx/raw

**Comment by jewbmx** [comment]
Heres a cfscrape update too thats going around. Might as well post it since people are asking for it anyways.   https://textuploader.com/1ours/raw

**Comment by bigpoppapaul** [comment]
> Heres a cfscrape update too thats going around. Might as well post it since people are asking for it anyways. https://textuploader.com/1ours/raw

Thanks for this. Worked beautifully.

**Comment by jewbmx** [comment]
Glad it worked lol, i just tossed it together and threw it up.
Think the actual fix was done by newt-sc

**Comment by reddit-reaper** [comment]
yeah that newt-sc fixed it last week, challenge issues are solved in that one you just need to add the final line for ex forks if you didnt already have it, though already been updated in OS as well. There's also going to be a new one soon once bugs are ironed out from venomous that will replace the old aronov version thats been updated over and over again. link for that is here

https://github.com/VeNoMouS/cloudscraper/blob/dev/cloudscraper/interpreters/native.py

**Comment by jewbmx** [comment]
Might be too high if ya spend 3 hours on a  rewrite of the sources.py module then click clear all, save, and close before realizing what you just did.

**Comment by jewbmx** [comment]
Think imma do a deep clean and rebuild of my github today. Sorry now for the soon to be activity spam lol, feel free to let me know if i cause any issues afterwards.

One goal is to rid us all of any bullshit tvaddons trash.

Altho what we all really need is a little script to scan for tvaddons trash and auto rewrite its files so they are easier to remove lmao
